bitbots_head_mover/config/head_config.yml

```diff
@@ -0,0 +1,212 @@
+move_head:
+  # Max values for the head position
+  max_pan: {
+    type: double_array,
+    default_value: [-2.35, 2.35],
+    description: "Max values for the head position (in radians)",
+    validation: {
+          fixed_size<>: 2
```

Comment:
Also add valid range

Reply:
Isn't max_pan already the valid range as it states the maximum position on both sides?

Reply:
I would assume that the maximum maximum value is one radian tho the right and to the left.

Reply:
then why are the defaults 2.34 > 2 radians on each side?